code/modules/mob/holder.dm

```diff
@@ -239,8 +239,6 @@ var/list/holder_mob_icon_cache = list()
 
 		post_scoop()
 
-		return success
-
```

Comment:
I am unsure why that return has been removed.
What error/message did you get there?

Reply:
Graater and small eror will be occur